src/app.rb

```diff
@@ -0,0 +1,64 @@
+require_relative 'storage'
+require_relative 'actions/utiles_helper'
+
+class App
+  def initialize
+    @store = Store.new
+    @B_utiles = BookUtiles.new(@store)
+    @M_utiles = MusicUtiles.new(@store)
+    @Movies_utiles = MovieUtiles.new(@store)
+    @G_utiles = GameUtiles.new(@store)
+  end
+
+  def sub_processor(option)
+    case option
+    when 1
+      @store.books.each do |book|
+        puts "id: #{book.id}, Publisher: #{book.publisher}, Publish Date: #{book.publish_date}, Cover State: #{book.cover_state}, label: #{book.label}"
+      end
+    when 2
+      @store.musics.each do |music|
+        puts "Title: #{music.title}, Publish Date: #{music.publish_date}, on Spotify: #{music.on_spotify}, genre: #{music.genre}"
+      end
```

Comment:
When I list the music_albums, the genre shows the ruby object instead of the data.

![image](https://user-images.githubusercontent.com/38088818/202693279-5702ea6a-82d5-4e31-8ca4-e635782603c6.png)

- Kindly look into it and make sure that genre text is displayed instead of the ruby object signature.